### PR TITLE
Clarify licensing

### PR DESCRIPTION
**Pull request by ssbarnea**
Fixes: #1188

### CODE REVIEW

**Review by gundalow — CHANGES_REQUESTED** [review]
@ssbarnea Thank you for updating this.

I'm just putting -1 till I get someone else to review this.

**Comment by gundalow** [comment]
@tadeboro Thank you for raising this (via https://github.com/ansible-community/ansible-lint/issues/1188) how does this working sound?

**Comment by ssbarnea** [comment]
@gundalow While this explanation does not have any legal value, it would worth contacting our legal team for feedback on licensing.

I am also quite curious to find about other Python packages that faced the same issue. I really doubt that **dynamic languages** that do not effectively embed any code from the GPL dependency are affected. In the end the string of bits, as source or package does not contain any GPL code and producing it (archiving) does not make use of any GPL code, so no lawyer would ever be able to win.

On the other hand, the reality is that you cannot make practical use of the project without also installing the GPL bits. 

AFAIK, that counts as weak-dependency  but it worth being documented in order to avoid confusions.

**Review by acozine — COMMENTED** [review]
I've tried to make the license statement easier to read and understand. Have I retained the correct meaning? 

Once we've agreed on the wording, it would be nice to add links to the licenses in the text.

**Comment by tadeboro** [comment]
The additions do solve the main issue I had: knowing what license applies to contributions. But because IANAL, I have no idea if this is enough to satisfy the GPLv3 requirements.

**Comment by abadger** [comment]
This is the clearest wording I've ever seen concerning this issue.  Well done ssbarnea and acozine!  

This looks like it agrees with the legal advice the lawyers have always given me about similar licensing situations.  +1 from me on that count.

File: setup.cfg [hunk 1]
```diff
@@ -88,16 +89,16 @@ console_scripts =
 
 # core will point to ansible-core>=2.11 as soon that is released
 community =
-  ansible>=2.10
+  ansible>=2.10  # GPLv3+
```
Comment (gundalow) [inline setup.cfg:92]: Tracking like this is a great idea, thank you

**Review by gundalow — APPROVED** [review]
Given @abadger's comment I'm changing my -1 to +1
Thank you all

**Pull request merged**